src/test/java/pages/AboutCompanyPage.java

```diff
@@ -0,0 +1,39 @@
+package pages;
+
+import elements.Text;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import util.WaitUtils;
+
+public class AboutCompanyPage extends BasePage {
+
+    @FindBy(xpath = "//p[2]/strong")
+    protected Text requisites;
+
+    @Override
+    public AboutCompanyPage openPage() {
+        driver.navigate().to(props.getKeyProperty("aboutURl"));
+        logger.debug("Navigation to the url...");
+        return this;
+    }
+
+    @Override
+    public AboutCompanyPage isPageOpened() {
+        try {
+            WaitUtils.waitForVisibility(requisites);
```

Comment:
implement methods for waiting inside element class, usage ex. `requisites. waitForVisibility()`